src/configBuilder.js

```diff
@@ -23,8 +25,13 @@ function tedious( config ) {
 		encrypt = false,
 		abortTransactionOnError = true } = config;
 	return {
-		userName: username,
-		password,
+		authentication: {
+			type: authenticationType,
+			options: {
+				userName: username,
+				password
+			}
+		},
 		server,
 		domain,
```

Comment:
`domain` needs to be moved up into the `authentication` object and the presence of that needs to change `authentication.type` from `default` to `ntlm`

Reply:
Amended with these changes.